Packs/illuminate/Integrations/Analyst1/Analyst1_description.md

```diff
@@ -0,0 +1,37 @@
+# Analyst1 Integration Configuration
```

Comment:
Per the style guide, the main heading for an integration description file should be a level 2 heading (`##`) and contain only the integration name.